charts/amoro/templates/amoro-configmap.yaml

```diff
@@ -0,0 +1,128 @@
+{{/*
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/}}
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Release.Name }}
+  labels:
+    {{- include "amoro.labels" . | nindent 4 }}
+data:
+  ## Helm chart provided Amoro configurations
+  config.yaml: |
+    ams:
+      admin-username: admin
+      admin-password: admin
+      server-bind-host: "0.0.0.0"
+      server-expose-host: "127.0.0.1"
```

Comment:
Using 127.0.0.1 will cause the RestCatalog URI generated by AMS to be accessible only from within the container. It may be necessary to expose the access address of AMS. However, this is not well adapted to non-IP address situations.

Reply:
Can I assume this is follow up done in other issuse? , currently we can use the **port-forward** command to access

Reply:
OK, I will follow this.